Article/Article.js

```diff
@@ -1,26 +1,29 @@
 // Because classes are not hoised you will need to start your code at the bottom of the page.  Look for the comment "START HERE"
 
 class Article {
-  constructor(domElement) {
-    // assign this.domElement to the passed in domElement
-    this.domElement;
-    // create a reference to the ".expandButton" class. 
-    this.expandButton;
-    // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
-    // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
-  }
-
-  expandArticle() {
-    // Using our reference to the domElement, toggle a class to expand or hide the article.
-
-  }
+    constructor(domElement) {
+        // Set Props
+        this.domElement = domElement;
+
+        this.expandButton = this.domElement.querySelector('.expandButton');
+
+        // Set Values
+        this.expandButton.innerHTML = 'expand';
+
+        // Event Handlers
+        this.expandButton.addEventListener('click', this.expandArticle.bind(this));
```

Comment:
`this.expandButton.addEventListener('click', () => this.expandArticle());` (Could also be written like this, doesn't matter just preference)